RELEASE.md

```diff
@@ -250,11 +252,9 @@ Then, the Release Manager releases and announces the new version:
          git push --tag origin self-hosted-x.x.x
          ```
 
--   [ ] 3.  Inform all stakeholders that the release is finished
-
     The final version will be `x.x.x`.
 
--   [ ] 4. Finally, the Technical Writer must [release a version of the documentation for the new Codacy Self-hosted version](https://github.com/codacy/docs/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-documentation).
+-   [ ] 3. Finally, the Technical Writer must [release a version of the documentation for the new Codacy Self-hosted version](https://github.com/codacy/docs/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-documentation).
```

Comment:
Extra indentation space just to keep the formatting consistent. :nerd_face: 

```suggestion
-   [ ] 3.  Finally, the Technical Writer must [release a version of the documentation for the new Codacy Self-hosted version](https://github.com/codacy/docs/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-documentation).
```

Reply:
@prcr we had to close this PR and open a new one (we did a mistake on branch name and were triggering the wrong CI build). This is open in #707 (we will apply your comments anyways)

Reply:
Yes, I also have to be careful not to open branches starting with `release` on the docs repository! :sweat_smile: